src/mixins/input.js

```diff
@@ -118,7 +118,7 @@ export default {
     },
     genIcon (type, defaultCallback = null) {
       const shouldClear = type === 'append' && this.clearable && this.isDirty
-      const icon = shouldClear ? 'clear' : this[`${type}Icon`]
+      const icon = shouldClear ? (this.clearableIcon || 'clear') : this[`${type}Icon`]
```

Comment:
Make 'clear' as a default value of `clearableIcon` prop

Reply:
Good catch btw

Reply:
I did 'clear' as default and just use:
const icon = shouldClear ? this.clearableIcon : this[`${type}Icon`]
But I don't like to duplicate two times 'clear' at v-text and v-select props :P

Reply:
The thing is that we're planning to add a possibility of setting global defaults, so instead of adding `clearableProp` to every `v-text-field` or `v-select` you could do it just one time in the config. Of course you could make your own components that wrap those two, but the behaviour should be consistent across all components.
Another option is to make a `clearable` mixin if you want to keep it DRY, but I don't think it's worth it

Reply:
I agree with you. I do not like redundancy. But in these two cases vuetify should be more flexible and not depend so much on Material Icons. At the moment it is the most viable option. If you want a "clearable" text field, you have to add that property to each textfield equally, unless you, as you said, make your own component.